browser/widevine/widevine_utils.h

```diff
@@ -33,4 +35,8 @@ void SetWidevineEnabled(bool opted_in);
 void MigrateWidevinePrefs(PrefService* prefs);
 void MigrateObsoleteWidevineLocalStatePrefs(PrefService* local_state);
 
+#if BUILDFLAG(IS_LINUX) && !defined(ARCH_CPU_X86_64)
+bool HasBundledWidevine();
```

Comment:
this method name made me confusing :) Maybe using `HasCopiedWideve()` more intuitive. WDYT?

Reply:
I see; What do you think of `IsWidevinePreinstalled`? I'm suggesting this alternative because `Copied` sounds very general to me.

Reply:
`IsWidevinePreinstalled()` looks good!

Reply:
Fixed in b1705af42913eaa12ef852d06dc47a008e6e73be.